### ISSUE TITLE
Enable tests in WearDataLayerAppHelperRoboTest

### ISSUE THREAD

**Issue by luizgrp** [google/horologist#2487]
It would require to update wear tiles to at least [alpha-05](https://developer.android.com/jetpack/androidx/releases/wear-tiles#1.5.0-alpha05).

Context: https://github.com/google/horologist/pull/2414/files#r1799948783

**Comment by oas004** [comment]
Is it okay to bump androids.wear.tiles to alpha-05 or above? I noticed that it is on 1.4.1 stable now :)

**Comment by oas004** [comment]
I tested this out with alpha05 and it seems to work other than the remove tile function (`clientUnderTest.sendOnTileRemovedEvent()`) .  Will try to investigate a bit further

**Comment by luizgrp** [comment]
> Is it okay to bump androids.wear.tiles to alpha-05 or above? I noticed that it is on 1.4.1 stable now :)

Yes, see [this](https://github.com/google/horologist?tab=readme-ov-file#maintained-versions). It's ok to bump to alpha in main, we try to keep stable for the release branches!

**Comment by luizgrp** [comment]
Fixed in https://github.com/google/horologist/pull/2718
